dtale/translations/cn.json

```diff
@@ -0,0 +1,150 @@
+{
+  "Charts": "图表",
+  "Back To Data": "Back To Data",
+  "Binning in Data Mining": "Binning in Data Mining",
+  "Equal Width": "Equal Width",
+  "Equal Frequency": "Equal Frequency",
+  "the bins are of equal sized ranges": "the bins are of equal sized ranges",
+  "the bins contain an equal amount of values": "the bins contain an equal amount of values",
+  "auto_load_msg": "By default, the chart builder will try to build a chart everytime you change your chart settings. If you toggle off 'Auto-Load' then you can change the settings as much as you like without triggering a chart build. In order to trigger a chart build with 'Auto-Load' off you must click the 'Load' button.",
+  "Projection": "Projection",
+  "To access additional styles enter a token here...": "To access additional styles enter a token here...",
+  "Change your token here...": "Change your token here...",
+  "Mapbox Access Token": "Mapbox Access Token",
+  "Style": "Style",
+  "ISO-3": "ISO-3",
+  "USA-states (use ISO)": "USA-states (use ISO)",
+  "country names (case-insensitive)": "country names (case-insensitive)",
+  "Load custom geojson into D-Tale": "Load custom geojson into D-Tale",
+  "View": "View",
+  "Location Mode": "Location Mode",
+  "Hourly": "Hourly",
+  "Hour": "小时",
+  "Weekday": "工作日",
+  "Weekly": "Weekly",
+  "Monthly": "Monthly",
+  "Quarterly": "Quarterly",
+  "Yearly": "Yearly",
+  "Default": "Default",
+  "Single": "Single",
+  "Multi": "Multi",
+  "Linear": "Linear",
+  "Log": "Log",
+  "ScatterGeo": "ScatterGeo",
+  "Detailed": "Detailed",
+  "Choropleth": "Choropleth",
+  "Enter pandas query": "Enter pandas query",
+  "Query": "Query",
+  "Load": "加载",
+  "Load a random percentage of rows from your dataset.": "Load a random percentage of rows from your dataset.",
+  "X": "X",
+  "Agg By": "Agg By",
+  "Default Index": "Default Index",
+  "Select Column(s)": "Select Column(s)",
+  "Y": "Y",
+  "Select Column": "Select Column",
+  "Z": "Z",
+  "Group": "Group",
+  "Select Group(s)": "Select Group(s)",
+  "Locations": "Locations",
+  "Lat": "Lat",
+  "Lon": "Lon",
+  "Scope": "Scope",
+  "Value": "Value",
+  "Open": "Open",
+  "High": "High",
+  "Low": "Low",
+  "Close": "Close",
+  "Labels": "标签",
+  "Aggregation": "集合",
+  "No Aggregation": "No Aggregation",
+  "Count": "计数",
+  "Unique Count": "独特的计数",,
```

Comment:
Just some extra commas here and after "Window" (I'm sure you already noticed this, but an extra set of eyes never hurts)

Reply:
Thanks! Saved me the research